server/container/guice/protocols/webadmin/src/main/java/org/apache/james/modules/server/WebAdminServerModule.java

```diff
@@ -138,6 +138,16 @@ public WebAdminConfiguration provideWebAdminConfiguration(FileSystem fileSystem,
         }
     }
 
+    @VisibleForTesting
+    ImmutableList<String> additionalRoutes(Configuration configurationFile) {
+        String rawString = configurationFile.getString("extensions.routes", "");
+
+        return ImmutableList.copyOf(Splitter.on(',')
```

Comment:
same as above: extract to local constant (note that I don't think mutualizing the two would be beneficial)

Reply:
This code path is only executed once.

Do we care? Sounds like early optimisation to me...